library/orientdb

```diff
@@ -4,13 +4,13 @@
 
 2.1.25: git://github.com/orientechnologies/orientdb-docker@8a9633c19fa9c53a5446b9b62997ff389813e785 2.1
 
-2.2.30: git://github.com/orientechnologies/orientdb-docker@0a229bb2c91c9115023c5010a3b46e2cf5c24925 2.2/x86_64/alpine
+2.2.30: git://github.com/orientechnologies/orientdb-docker@98c76824843b8c44944a14c5bd62af7e358af670 2.2/x86_64/alpine
 
-2.2.30-spatial: git://github.com/orientechnologies/orientdb-docker@0a229bb2c91c9115023c5010a3b46e2cf5c24925 2.2-spatial/x86_64/alpine
+2.2.30-spatial: git://github.com/orientechnologies/orientdb-docker@98c76824843b8c44944a14c5bd62af7e358af670 2.2-spatial/x86_64/alpine
```

Comment:
The tags should also be updated: `2.2.30` and `2.2.30-spatial`.

Reply:
Sorry, my bad, I just pushed a fix

Thanks

Luigi